### ISSUE TITLE
Example in the settings page has bad JSON

### ISSUE THREAD

**Issue by danielocdh** [moodle-an-hochschulen/moodle-atto_styles#20]
Example in the settings page has bad JSON:
```
For example:
{
    "title": "Blue box",
    "type": "block",
    "classes": "box blue"
    "preview": true
}
```
It also would be nice to have some kind of validation, in case the user inputs invalid JSON

**Comment by uaiblaine** [comment]
For example:
{
    "title": "Blue box",
    "type": "block",
    "classes": "box blue"**,** (missing the comma)
    "preview": true
}

**Comment by danielocdh** [comment]
@andersonblaine I know now but at first I trusted the plugin example and copied+pasted. The json with error is the example shown by the plugin in its settings. I know it is unlikely it gets fixed so hopefully this issue helps someone else with the same problem realize it

**Comment by Kathrin84** [comment]
Hello @danielocdh,

thanks for raising this issue and sorry for the late response.

We don't know why you think that 

> it is unlikely it gets fixed 

We give our best to maintain all of our plugins with our limited resources. Sometimes it can take some time for us to do the things, but we are always eager to deliver a robust and well working plugin.

So many thanks for pointing us at this flaw and proposing to add a JSON validation.

We implemented it:
https://github.com/moodleuulm/moodle-atto_styles/commit/addc0c0a06854b5947fc54e6f1ed2211e52b0725

It will be available in the next v3.8-r2 version of the plugin very soon.

So as this is fixed and improved, I'll close this issue at this point.

Best, Kathrin
